docs/permissions/getting-started.md

```diff
@@ -77,10 +77,7 @@ export default async function createPlugin(
     logger: env.logger,
     discovery: env.discovery,
     policy: new TestPermissionPolicy(),
-    identity: IdentityClient.create({
```

Comment:
If you want to holder off until 1.6.0 is released before this change goes out that's fine by me